.concourse/concourse.yaml

```diff
@@ -103,22 +103,22 @@ jobs:
             export KUBECONFIG=/tmp/kubeconfig.yaml
             istio-installer/bin/test.sh --skip-cleanup src/github.com/istio/istio/
 
-  - name: install-1.1.0
+  - name: install-old-master # we probably want 1.1 here, but we would need an old installer
     serial_groups: [ cluster ]
     plan:
     - get: 24h
       trigger: true
-      passed: [test-after-install]
+      passed: [ test-after-install ]
     - get: istio-installer
```

Comment:
If we use istio-installer master to install a fixed, older istio version from which we can do an upgrade, I suggest using a fixed older version of the installer as well. Otherwise we have to ensure that the installer master is still compatible with this fixed, older version.